.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
 install:
   - pip install git+https://github.com/FOSSEE/online_test.git#egg=yaksh-0.1
   - pip install -q Django==$DJANGO --use-mirrors
+  - pip install -q pytz
```

Comment:
Please check the version that you have installed while testing the feature and use the same one here,
```pip install -q pytz==<version>```
(instead of <version> use the version number that you installed)